src/cli/README_NETDATA.md

```diff
@@ -0,0 +1,214 @@
+
+   ```bash
+   dataset commit active
+   Done
+   ```
+
+3. Enable Thread interface
+
+   ```bash
+   > ifconfig up
+   Done
+   > thread start
+   Done
+   ```
+
+4. Observe IPv6 addresses assigned to the Thread inteface.
+
+   ```bash
+   > ipaddr
+   fd3d:b50b:f96d:722d:0:ff:fe00:fc00
+   fd3d:b50b:f96d:722d:0:ff:fe00:dc00
+   fd3d:b50b:f96d:722d:393c:462d:e8d2:db32
+   fe80:0:0:0:a40b:197f:593d:ca61
+   Done
+   ```
+
+5. Register an IPv6 prefix assigned to the Thread network.
+
+   ```bash
+   > prefix add fd00:dead:beef:cafe::/64 paros
+   Done
+   > netdata register
+   Done
+   ```
+
+6. Observe Thread Network Data.
+
+   ```bash
+   > netdata show
+   Prefixes:
+   fd00:dead:beef:cafe::/64 paros med dc00
+   Routes:
+   Services:
+   Done
+   ```
+
+7. Observe IPv6 addresses assigned to the Thread interface.
+
+   ```bash
+   > ipaddr
+   fd00:dead:beef:cafe:4da8:5234:4aa2:4cfa
+   fd3d:b50b:f96d:722d:0:ff:fe00:fc00
+   fd3d:b50b:f96d:722d:0:ff:fe00:dc00
+   fd3d:b50b:f96d:722d:393c:462d:e8d2:db32
+   fe80:0:0:0:a40b:197f:593d:ca61
+   Done
+   ```
+
+### Attach to Existing Network
+
+Only the Master Key is required for a device to attach to a Thread network.
+
+While not required, specifying the channel avoids the need to search across multiple channels, improving both latency and efficiency of the attach process.
+
+After the device successfully attaches to a Thread network, the device will retrieve the complete Active Operational Dataset.
+
+1. Create a partial Active Operational Dataset.
+
+   ```bash
+   > dataset masterkey dfd34f0f05cad978ec4e32b0413038ff
+   Done
+   > dataset commit active
+   Done
+   ```
+
+2. Enable Thread interface.
+
+   ```bash
+   > ifconfig up
+   Done
+   > thread start
+   Done
+   ```
+
+3. After attaching, observe Thread Network Data.
+
+   ```bash
+   > netdata show
+   Prefixes:
+   fd00:dead:beef:cafe::/64 paros med dc00
+   Routes:
+   Services:
+   Done
+   ```
+
+4. Observe IPv6 addresses assigned to the Thread interface.
+
+   ```bash
+   > ipaddr
+   fd00:dead:beef:cafe:4da8:5234:4aa2:4cfa
+   fd3d:b50b:f96d:722d:0:ff:fe00:fc00
+   fd3d:b50b:f96d:722d:0:ff:fe00:dc00
+   fd3d:b50b:f96d:722d:393c:462d:e8d2:db32
+   fe80:0:0:0:a40b:197f:593d:ca61
+   Done
+   ```
+
+## Command List
+
+- [help](#help)
+- [register](#register)
+- [show](#show)
+- [steeringdata](#steeringdata-check-eui64discerner)
+
+## Command Details
+
+### help
+
+Usage: `netdata help`
+
+Print netdata help menu.
+
+```bash
+> netdata help
+help
+register
+show
+steeringdata
+Done
+```
+
+### register
+
+Usage: `dataset register`
+
+Register configured prefixes, routes, and services with the Leader.
+
+```bash
+> netdata register
+Done
+```
+
+### show
+
+Usage: `dataset show [binary]`
+
+Print Network Data received from the Leader.
+
+```bash
+> netdata show
+Prefixes:
+fd00:dead:beef:cafe::/64 paros med dc00
+Routes:
+Services:
+Done
+```
+
+Print Network Data as hex-encoded TLVs.
+
+```bash
+> netdata show binary
```

Comment:
```suggestion
> netdata show -x
```

`ping` command uses `-x` argument to send binary data in hex. Should be be consistent throughout the CLI commands?

Reply:
I think it depends on the version of ping: https://linux.die.net/man/8/ping

I was simply following what was done for [dataset active](https://github.com/openthread/openthread/blob/master/src/cli/README_DATASET.md#active). However, I am OK to use something else if that's what others prefer.

Reply:
I do not see the "-x" option for OT CLI ping. However, I do see a "-x" option for OT CLI UDP example.

Reply:
Sorry. I meant `udp send`.